### PR TITLE
server token handle

### LINKED ISSUE #8: Handle server token
we need to change the way we handle the token and send it as an object to the front-end instead of putting it in the cookies, 

cause then it allow the front-end to set it as a cookie, and every time we send a request to the server it will look for the token in the headers to know witch user is sending the request, in addition it will secure the website.

### PR DESCRIPTION
**Pull request by FrancisA2000**
close #8 

explanation of this PR in the issue link

### CODE REVIEW

File: src/middlewares/authCheck.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const jwt = require("jsonwebtoken");
+
+module.exports = authCheck = (req, res, next) => {
+    const token =req.headers.authorization
+    const newToken = token.slice(7,token.length)
```
Comment (Karyum) [inline src/middlewares/authCheck.js]: if `authorization` header doesn't exist, it looks like your server will crash

Comment (Karyum) [reply src/middlewares/authCheck.js]: also slicing the token is weird im guessing it's something to do with `ZoIiwiY`?

**Review by Karyum — CHANGES_REQUESTED** [review]
remove all the `JSON.stringify` when you do `res.send` it's pointless since express does it for you

**Comment by Karyum** [comment]
@FrancisA2000 again issue and description for this PR

**Comment by Karyum** [comment]
@FrancisA2000 assign me when this branch is ready

**Comment by Karyum** [comment]
@FrancisA2000 issue and description, you know next projects are client projects, right now with these PRs im being very nice but i can't promise the same behavior next projects.

**Comment by FrancisA2000** [comment]
This PR changes the way we handle the token and send it as an object to the front-end instead of putting it in the cookies, In this feature, it allow the front-end to set it as a cookie, and every time we send a request to the server it will look for the token in the headers to know witch user is sending the request, in addition it will secure the website.

**Comment by Karyum** [comment]
@FrancisA2000 i made an issue and modified the pull request description, is it correct or do i need to change something ? i hope this helps you open issues in the future before working on a task and making a PR

**Comment by FrancisA2000** [comment]
aa ok cool,
thanks,

**Review by Karyum — APPROVED** [review]
looks great

**Pull request merged**